Dockerfile

```diff
@@ -1,58 +1,223 @@
-FROM ubuntu:16.04
-
-RUN apt-get update && \
-    apt-get -y install sudo && \
-    sudo apt-get update -qq && \
-    apt-get install -y python3 \
-                       python3-pip \
-                       python3-dev \
-                       build-essential \
-                       software-properties-common \
-                       openjdk-8-jdk \
-                       git \
-                       wget && \
-    sudo add-apt-repository ppa:openjdk-r/ppa && \
-         apt-get update -qq && \
-         apt-get install -y openjdk-8-jdk
+# Generated by: Neurodocker version 0.7.0+0.gdc97516.dirty
+# Latest release: Neurodocker version 0.8.0
+# Timestamp: 2022/03/30 12:05:02 UTC
+# 
+# Thank you for using Neurodocker. If you discover any issues
+# or ways to improve this software, please submit an issue or
+# pull request on our GitHub repository:
+# 
+#     https://github.com/ReproNim/neurodocker
+
+FROM debian:stretch-slim
```

Comment:
There exists pre-made miniconda docker images which can simplify this container.

https://hub.docker.com/r/condaforge/miniforge3

Reply:
Or even better, mamba
https://hub.docker.com/r/condaforge/mambaforge

Reply:
Sweet. Thanks!!